packages/core/data-transfer/src/strapi/providers/local-source/assets.ts

```diff
@@ -29,7 +29,7 @@ function getFileStream(
 
       if (res.body) {
         // pipe the image data
-        Readable.fromWeb(new webStream.ReadableStream(res.body)).pipe(readableStream);
+        Readable.fromWeb(res.body as any).pipe(readableStream);
```

Comment:
The cast to `any` means that `res.body` is not be fully compatible with `Readable.fromWeb`, are we sure we can do this without breaking anything? What's the actual type of `res.body`?

Reply:
it is `ReadableStream<Uint8Array<ArrayBufferLike>>`
and the method expects `ReadableStream<any>`
yeah I mean it was broken so it can't break anything technically

Reply:
I've checked some documentation, and it seems it's mainly a type mismatch due to node/dom implementation but they should indeed be compatible.

Could you at least remove the any and replace it with something like
```suggestion
        Readable.fromWeb(res.body as webStream.ReadableStream<Uint8Array>).pipe(readableStream);
```

Reply:
That doesn't seem to work either. I think I would prefer a `@ts-expect-error` though over `as any`.

From what I can tell this is a known type incompatibility in a usage where the objects themselves are actually compatible. Then at some point hopefully the types will get fixed and we can remove it.

Reply:
>That doesn't seem to work either.

What do you mean by it doesn't work either ? :thinking: 
![image](https://github.com/user-attachments/assets/423fdcae-2351-4bf4-90b4-5c270c426dec)
![image](https://github.com/user-attachments/assets/648f3f7c-6bf7-49b9-b80d-92d54b78d967)
![image](https://github.com/user-attachments/assets/7cb2750e-7eb2-4baa-a1b3-61ab83171d06)

> From what I can tell this is a known type incompatibility in a usage where the objects themselves are actually compatible. Then at some point hopefully the types will get fixed and we can remove it.

yep

Reply:
It does work I changed the casting to webStream.ReadableStream<Uint8Array> that seems to satisfy it

Reply:
Ahh, yeah, it does work, nevermind 😆